.github/pull_request_template.md

```diff
@@ -10,3 +8,9 @@ A clear and concise description of what the bug or new feature is.
 - Add [Tests](https://github.com/lvgl/lvgl/blob/master/tests/README.md) if applicable.
 - If you added new options to `lv_conf_template.h` run [lv_conf_internal_gen.py](https://github.com/lvgl/lvgl/blob/master/scripts/lv_conf_internal_gen.py) and update [Kconfig](https://github.com/lvgl/lvgl/blob/master/Kconfig).
 - Run `scripts/code-format.py` ([astyle](http://astyle.sourceforge.net/install.html) needs to be installed) and follow the [Code Conventions](https://docs.lvgl.io/master/CODING_STYLE.html)
+- Mark the Pull request as [Draft](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-stage-of-a-pull-request) while you are working on the first version, and mark is as _Ready_ when it's ready for review.
```

Comment:
Well, I think the process of PR review should not be written in the PR template, instead the template should contain a **check-list** that maintainer should go through when creating PR (tests added, document added, configs changed, etc...). The process of PR review should be written in the contributing document.
But of course you can reference the Github PR process as a link.

In the template there could be also a line mentioning related issues, and/or other PR dependencies.

Reply:
Well, if you want to keep these 2 lines in PR template, then those should be in a separate section, title could be: "PR process".
These 2 are not checks, but describe the process

Reply:
What about renaming "Checkpoints" to "Notes"? It will save the space of one more heading and the extra spaces around it

Reply:
that’s a good idea!

Reply:
Updated